ui/DesignSystem/Primitive/Icon/Cross.svelte

```diff
@@ -1,50 +1,63 @@
 <script>
   export let style = null;
-  export let size = "small"; // normal | "big"
+  export let variant = "small"; // small | medium | big | filled | outline
 </script>
 
-{#if size === 'small'}
-  <svg
-    {style}
-    width="24"
-    height="24"
-    viewBox="0 0 24 24"
-    fill="var(--color-foreground-level-5)"
-    xmlns="http://www.w3.org/2000/svg">
+<svg
+  {style}
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="var(--color-foreground-level-5)"
+  xmlns="http://www.w3.org/2000/svg">
+  {#if variant === 'small'}
     <path
       fill-rule="evenodd"
       clip-rule="evenodd"
-      d="M7.29289 7.29289C7.68342 6.90237 8.31658 6.90237 8.70711
-      7.29289L16.7071 15.2929C17.0976 15.6834 17.0976 16.3166 16.7071
-      16.7071C16.3166 17.0976 15.6834 17.0976 15.2929 16.7071L7.29289
-      8.70711C6.90237 8.31658 6.90237 7.68342 7.29289 7.29289Z" />
+      d="M7.29289 7.29289C7.68342 6.90237 8.31658 6.90237 8.70711 7.29289L12
+      10.5858L15.2929 7.29289C15.6834 6.90237 16.3166 6.90237 16.7071
+      7.29289C17.0976 7.68342 17.0976 8.31658 16.7071 8.70711L13.4142 12L16.7071
+      15.2929C17.0976 15.6834 17.0976 16.3166 16.7071 16.7071C16.3166 17.0976
+      15.6834 17.0976 15.2929 16.7071L12 13.4142L8.70711 16.7071C8.31658 17.0976
+      7.68342 17.0976 7.29289 16.7071C6.90237 16.3166 6.90237 15.6834 7.29289
+      15.2929L10.5858 12L7.29289 8.70711C6.90237 8.31658 6.90237 7.68342 7.29289
+      7.29289Z" />
+  {:else if variant === 'medium'}
     <path
       fill-rule="evenodd"
       clip-rule="evenodd"
-      d="M7.29292 16.7071C6.9024 16.3166 6.9024 15.6834 7.29292 15.2929L15.2929
-      7.29289C15.6834 6.90237 16.3166 6.90237 16.7071 7.29289C17.0977 7.68342
-      17.0977 8.31658 16.7071 8.70711L8.70714 16.7071C8.31661 17.0976 7.68345
-      17.0976 7.29292 16.7071Z" />
-  </svg>
-{:else if (size = 'big')}
-  <svg
-    {style}
-    on:click
```

Comment:
These directives are important for events to bubble up to components who use these icons. It's the reason the tests in your last commit fail, check [this docs section](https://svelte.dev/docs#on_element_event) and especially the part on fowarding. I added this to all icons in 49dff34.

Reply:
Cool